en_US/advanced/cluster.md

```diff
@@ -58,6 +58,47 @@ Cookies are used for interconnection authentication between Erlang nodes. A cook
 
 See <http://erlang.org/doc/reference_manual/distributed.html> for details.
 
+### Using TLS for backplane connections
+It is possible to enable TLS encryption for the backplane connections. It comes at the cost of increased CPU load, though.
+
+1. Generate a CA-signed certificate using `openssl` tool:
+  ```
+  # Create self-signed root CA:
+  openssl req -nodes -x509 -sha256 -days 1825 -newkey rsa:2048 -keyout rootCA.key -out rootCA.crt -subj "/O=LocalOrg/CN=LocalOrg-Root-CA"
```

Comment:
use `.pem` for keys and certificates (like existing files in `etc/certs`)

Reply:
Done